spec/examples/hourly_rule_spec.rb

```diff
@@ -42,10 +42,10 @@ module IceCube
         schedule = Schedule.new(t0 = Time.local(2013, 11, 3, 0, 0, 0))
         schedule.add_recurrence_rule Rule.hourly
         schedule.first(4).should == [
-          Time.local(2013, 11, 3, 0, 0, 0),             # -0700
-          Time.local(2013, 11, 3, 1, 0, 0) - ONE_HOUR,  # -0700
-          Time.local(2013, 11, 3, 1, 0, 0),             # -0800
-          Time.local(2013, 11, 3, 2, 0, 0),             # -0800
+          Time.new(2013, 11, 3, 0, 0, 0, '-07:00'),
```

Comment:
I'm pretty sure this test as written relied on incorrect behavior of `Time::local` - unless I'm really confused (always possible) 1:00 AM on November 3rd, 2013, is still in DST, as DST doesn't end until 2:00 AM. I've filed this issue against Ruby as https://bugs.ruby-lang.org/issues/12147. At any rate, I think this rewritten test is a bit of an improvement, as it makes the offset an explicit part of the test.